app/main.ipynb

```diff
@@ -671,8 +880,8 @@
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T14:20:27.503470Z",
-     "start_time": "2023-07-03T14:20:27.458680Z"
+     "end_time": "2024-04-27T22:47:50.635125Z",
+     "start_time": "2024-04-27T22:47:50.617570500Z"
```

Comment:
Cannot see preview of file because of this error. Add "execution_count" after the last "cell_type": "markdown"
<img width="670" alt="image" src="https://github.com/mate-academy/py-numpy/assets/61380547/cadf84d3-6af7-4fb8-9e75-cfdf1181a64d">